cli/src/pcluster/config/cluster_config.py

```diff
@@ -1429,6 +1452,7 @@ def _register_storage_validators(self):  # noqa: C901 FIXME: function too comple
             )
 
             existing_fsx = set()
+            existing_fsx_file_cache_id = set()
```

Comment:
Why FileCache needs a separated set while any other FSx storage insist on the same set?
If the reason is that you want to apply a validator only to the existing FC and not to all FSx storage, then I would have the following follow up questions:
1. why a validator would be necessary only for FC but not for the other FSx storage types?
2. what about having a unique set and filtering FC within this set, e.g. [fc_id for fc_id in existing_fsx if fc_id.starts_with("fc-")] rather than splitting the concepts.

Reply:
I am not removing validation for the other storages. There are a few paramters that are different for file systems and file cache, but I can incorporate something to make it simpler.